testsuite/src/Test/Tasty/Clash.hs

```diff
@@ -724,7 +724,7 @@ outputTest env targets extraClashArgs extraGhcArgs modName funcName path =
   testGroup testName
     [outputTest' env target extraClashArgs extraGhcArgs modName funcName path' | target <- targets]
 
-netlistTest'
+clashLibTest'
```

Comment:
This would still operate on Netlist right? Or are you planning on extending this?

Reply:
Still operates on netlist, but can be reused for core. I changed the name because it really applies to any test that needs to run parts of `clash-lib`